pkg/controller/disruption/disruption.go

```diff
@@ -557,7 +633,9 @@ func (dc *DisruptionController) failSafe(pdb *policy.PodDisruptionBudget) error
 	return dc.getUpdater()(&newPdb)
 }
 
-func (dc *DisruptionController) updatePdbSpec(pdb *policy.PodDisruptionBudget, currentHealthy, desiredHealthy, expectedCount int32) error {
+func (dc *DisruptionController) updatePdbSpec(pdb *policy.PodDisruptionBudget, currentHealthy, desiredHealthy, expectedCount int32,
```

Comment:
Can you rename this function to something like `updatePdbStatus`? Currently it's misleading since there is no spec update involved.